.travis.yml

```diff
@@ -9,10 +9,12 @@ cache:
 
 env:
   global:
-    - ANSIBLE_HOST_KEY_CHECKING=False
-    - PIP_DOWNLOAD_CACHE=$HOME/.cache/pip
-    - OC_BINARY_URL=https://mirror.openshift.com/pub/openshift-v3/clients/3.10.45/linux/oc.tar.gz
+    - ANSIBLE_HOST_KEY_CHECKING="False"
+    - PIP_DOWNLOAD_CACHE="$HOME/.cache/pip"
+    - OC_BINARY_URL="https://mirror.openshift.com/pub/openshift-v3/clients/3.11.133/linux/oc.tar.gz"
     - ANSIBLE_VERSION="2.7"
+    - OCP_CLUSTER_URL="https://console.ci-1.cop.rht-labs.com"
```

Comment:
Would prefer to not add our URLs in public github, so let's see if we can find a way to pass this in without keeping it in the repo(s).